example/imports.yaml

```diff
@@ -1,32 +1,108 @@
+cluster: |
+  apiVersion: v1
+  kind: Config
+# ... <please insert your kubeconfig here>
+
 hostingCluster:
-  kubeconfig: |
-    apiVersion: v1
-    kind: Config
-#   ... <please insert your kubeconfig here>
-  namespace: garden-123
+  namespace: garden-4
   infrastructureProvider: gcp # aws|gcp|alicloud
 
 virtualGarden:
+  deleteNamespace: true
+
   etcd:
     storageClassName: my-virtual-garden-storage-class
     backup:
       infrastructureProvider: gcp # aws|gcp|alicloud
-      region: europe-west5
+      region: europe-west555
       bucketName: my-virtual-garden-backup
-      credentialsRef: mycredentials
-    hvpa: {}
+      credentials:
+        data:
+          serviceaccount.json: |
+            {
+              "project_id": "test-project",
+              "type": "test-type"
+            }
+
+    handleETCDPersistentVolumes: true
+    hvpaEnabled: true
 
   kubeAPIServer:
-  # exposure:
-  #   sni:
-  #     hostnames:
-  #     - foo.com
-  #     dnsClass: my-class
-  #     ttl: 120
+    replicas: 1
+    sni:
+      hostname: api.dev.gardener.cloud.sap
+      dnsClass: gardendns
+      ttl: 120
+      secretName: gardener-dashboard-tls
+    dnsAccessDomain: my.dns.domain
+
+    gardenerControlplane:
```

Comment:
What has the gardener control plane to do with the `kubeAPIServer` configuration?